### DISCUSSION TITLE
Mobile broadcast support

### DISCUSSION THREAD

**Discussion by remram44** [owncast/owncast#664] [Q&A] [answered]
I have no problem streaming from OBS, however I could not stream from my Android phone.

I tried [Larix Broadcaster](https://play.google.com/store/apps/details?id=com.wmspanel.larix_broadcaster): it has not place to put in a stream key. `rtmp://myserver:9135/live/streamkey` didn't work, and `rtmp://myserver:9135/streamkey` is refused by the app.

I tried [RTMP Camera](https://play.google.com/store/apps/details?id=com.miv.rtmpcamera): again, no place to put a stream key, both `myserver:9135/live/streamkey` and `myserver:9135/streamkey` gave me connection errors.

Before I turn off my PeerTube and do all the tests again on the default port (1935), did anyone manage to do this successfully? With any Android app?

What is the correct URL for apps that don't expect a "stream key", `/live/streamkey` or just `/streamkey`?

Thanks

**Comment by gabek** [comment, marked as answer]
`rtmp://myserver/live/yourstreamkey` should work for any app that doesn't natively have support for a stream key.  I've actually used Larix, but on iOS, with no issues in the past.  So hopefully it works for you as well on Android!

Feel free to use our test server if you want to test it on 1935 somewhere without having to change your own server: https://broadcast.owncast.online/

**Reply by remram44** [reply]
Thanks I got it to work! It was an IPv6 error (not necessarily OwnCast's fault, I suspect Docker). Putting the IPv4 of my server instead of my hostname solved the problem, and I was able to broadcast with Larix Broadcaster. RTMP Camera gives a (possibly unrelated) Java "array out of bounds" exception.
